data/ship builders jobs.txt

```diff
@@ -0,0 +1,132 @@
+	description "Escort the ship with the new cargo-systems to <destination> to allow it to deliver cooked fish dishes to an archeology students party."
+	source "Skymoot"
+	destination "Zug"
+	landing
+	to offer
+		has "Escort Hauler prototype to formal dinner: offered"	
+	npc accompany
+		government "Merchant"
+		ship "Hauler III" "SB-Labs Cargo Cooker"
+	on enter
+		dialog
+			`As soon as you take off you receive an angry voice message over a short-range inter-ship communications channel: "Why were you trying to contact me over radio! Does everybody here need to know that we are flying around here???`
+			`Let's go to <destination>." You wonder why the destination changed.`
+	on complete
+		dialog
+			`This time the Hauler prototype appears to be landing right behind you, but the ship executes a go-around just before touching the landing pad and flies away at full thrust.`
+			`You receive a short text-message over the short-range inter-ship communiation channel: "The colleagues at the shipyard cannot see us like this!"`
+
+mission "Escort Hauler prototype to factory"
+	description "Escort the ship with the new cargo-systems to <destination> to allow it to deliver overcooked fish to a floating catfood factory."
+	source "Zug"
+	destination "Mere"
+	landing
+	to offer
+		has "Escort Hauler prototype to student party: offered"
+	npc accompany
+		government "Merchant"
+		ship "Hauler III" "SB-Labs Cargo Processor"
+	on complete
+		dialog
+			`The Hauler VI prototype descends at full thrust and lands way ahead of you on a landing pad from a floating catfood factory.`
+			`Your infrared sensors show that the ship emits a lot of heat, but it not from the heavily used thrusters but from the cargo-hold.`
+			`Ms. Oxygenpocket steps out of the ship and says: "This is no good for living creatures, we go to <destination>."`
+			`She enters the prototype and takes off before you have time to respond. Apparently there is something seriously wrong with the cargo that was formerly known as sushi.`
+
+mission "Escort Hauler prototype to waste plant"
+	description "Escort the ship with the new cargo-systems to <destination> to allow it to deliver charred fish wasteproducts to a waste processing plant."
+	destination "Longjump"
+	landing
+	on offer
+	to offer
+		has "Escort Hauler prototype to factory: offered"
+	on complete
+		payment 200000
+		dialog
+			`The Hauler prototype lands somewhat uncontrolled near a waste processing plant. The cargo hold is glowing red hot and some areas in the cargo hold appear to be on fire.`
+			`Ms. Oxygenpocket disembarks, walks to you and apologises for the difficult trip. She explains that the cooling systems failed causing the sushi to get cooked, get overcooked and finally on fire.`
+			`You get paid <payment> and she asks you not to talk about this escort job to anybody. You leave the docks quicky in order to escape the horrible smell of overheated fish that is comming from Ms. Oxygenpocket and her ship.`
```

Comment:
The word "get" should be avoided in general, because it has a very informal connotation, which is not proper for Endless Sky narration. This should instead read, "You are paid `<payment>`."

"Quicky" -> "quickly."